### DISCUSSION TITLE
GitHub codes not displaying properly

### DISCUSSION THREAD

**Discussion by x17green** [community/community#62177] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![Screenshot_2023-07-30_023425](https://github.com/community/community/assets/108209156/4b6d39b5-6613-4c12-8dd7-45f4f7d3d9e5)

Why do my GitHub codes diplay like this?

Check attached

**Comment by Calc-oholic** [comment, marked as answer]
Try clearing your cookies and then refreshing.

Steps (for Chrome):
1. Click at the left of the address bar on the lock icon
2. Click cookies and site data
3. Click "manage cookies and site data"
4. Click on the trash can next to Github
5. reload the page

Let me know if it works!

**Reply by x17green** [reply]
Okay, I'll try it out and let you know.

**Reply by x17green** [reply]
Thanks, @Calc-oholic It did work, and an extension was blocking it, so I removed it and worked.

**Comment by cehfisher** [comment]
Hi @x17-Green - It looks like your issue was mistakenly added to the accessibility discussion board, but it was resolved. @queenofcorgis could you please move this to the correct category and close it out? Thanks!
